src/uct/tcp/tcp.h

```diff
@@ -360,6 +358,7 @@ typedef struct uct_tcp_iface_config {
 
 
 extern uct_component_t uct_tcp_component;
+extern const uint64_t uct_tcp_cm_magic_number;
```

Comment:
the magic number could still be a `#define` but it has to be real 64 bit value
for example 0xCAFEBABE12345678lu